deploy.sh

```diff
@@ -33,7 +33,7 @@ echo -e "\tCriIdentifier SSM key ${GREEN}$cri_identifier${NOCOLOR}"
 
 ./gradlew clean
 sam validate -t infrastructure/lambda/template.yaml
-sam build -t infrastructure/lambda/template.yaml
+sam build -t infrastructure/lambda/template.yaml --beta-features
```

Comment:
Do we need to commit this @jkunle  and @gokhakan

Reply:
No we shouldnt need this at all

Reply:
I am removing this. It works without.